.slather.yml

```diff
@@ -5,3 +5,6 @@ scheme: ios-base-develop
 binary_basename: ios-base-Debug
 ignore:
   - Pods/*
+  - ios-base/Extensions/UIImageExtension.swift
+  - R.generated.swift
+  - ios-base/AppDelegate.swift
```

Comment:
Why ignoring those files here?

Reply:
They are taken into account for test coverage and they negatively contribute to it, even when they aren't tested and testing those adds no value.
i.e UIImageExtension has a method to generate test images for the demo sign up.
R.generated is autogenerated and it should be tested in the corresponding pod.